unconference/LMIC-know-how.md

```diff
@@ -0,0 +1,200 @@
+    * Connecting with your institution and learning about procedures
+    * Finding or learning if you need to create an organization to support you (fiscal sponsorship)
+* Finding supporters that will give you help/feedback for your proposal
+* Templates
+    * Collaborator recommendation letters 
+    * CVs
+
+*Module 2 learning objectives:* 
+At the end of this module, learners should be able to:
+* Identify grant opportunities for which they are eligible
+* Identify key collaborators or stakeholders that need to be involved
+* Understand the basic requirements you need to meet for a grant application
+* Decypher the RFA specifics and translate them into actionable application tasks
+
+
+### Module 3: Preparing a proposal
+
+* Preparing a proposal
+    * Sections of a proposal
+    * Research narrative
+    * Diversity, equity, and inclusion (DEI) statement
+* Deliverables and outcomes
+* Letters of recommendation/support
+    * Formatting
+    * Aims/objectives/goals 
+    * Questions/hypotheses
+    * Methods
+    * Deliverables and success metrics
+    * Citation/references
+    * Supporting materials 
+    * Lexicon (how to handle acronyms)
+* Timelines
+* What makes a good proposal
+    * Consistency
+    * Concise but clear (not too specific or too general)
+* Resources at your affiliated institution or organization 
+* Institutional approval
+* Listing your collaborators
+* Steps in the application process
+    * Letters of intent
+    * Full applications
+    * Interviews
+* Seek peer feedback
+    * From folks within your field and beyond your field - until you are proficient enough to be your own reviewer
+
+*Module 3 learning objectives:* 
+At the end of this module, learners should be able to:
+* Prepare a template to cover text-based grant proposal documents, tailored to the needs of a grant funder and/or grant call;
+* Define a scope of work alongside collaborators, and incorporate it into a grant narrative;
+* Identify and comply to the learner’s organizational requirements for grantmaking;
+* Create detailed and realistic project timelines and staffing estimates
+
+
+### Module 4: Budgeting
+
+* Creating your own budget, using a provided template, or using an institutional template
+* How to charge for staff time (salary, contractors, fringe/benefits), adjust for inflation
+* Overheads and indirect costs
+* Budget narrative or justifying your budget
+* Current and pending funding, non-grant income
+* The problems of under budgeting 
+
+*Module 4 learning objectives:*
+At the end of this module, learners should be able to:
+* Create a basic budget with categories and formatting suitable for submission to a funder
+* Avoid under-budgeting by proposing adequate budget for inflation and contingencies
+* Describe current and pending support in a way that is meaningful for grant review 
+* Understand indirect costs, including what your organization requires and what a funder allows
+
+
+### Module 5: You got the grant - now what?
```

Comment:
probably add items for
* something changes, and I need to change the project - how do I do this?
* possibly extending the project (For NSF, two options are a no-cost extension that extends the project term with no additional funding; a supplement that adds funding for new work related to the project but doesn't extend the project term.) Some projects also can be renewed, which isn't the same as a new proposal, but is related.